### PR TITLE
test(sbb-tab-group, sbb-tab-label): implement visual regression tests

### PR DESCRIPTION
**Pull request by MarioCastigliano**

### CODE REVIEW

File: src/elements/tabs/tab-label/tab-label.visual.spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { html } from 'lit';
```
Comment (DavideMininni-Fincons) [inline src/elements/tabs/tab-label/tab-label.visual.spec.ts:1]: I don't know if could be truly useful, but i would like to have other cases covered, eg. without amount, without icon, without both, slotted icon, slotted amount, long label etc; basically the same cases covered in stories.

File: src/elements/tabs/tab-group/tab-group.visual.spec.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        }),
+      );
+    });
+
+    it(
+      visualDiffFocus.name,
+      visualDiffFocus.with(async (setup) => {
+        await setup.withFixture(html`
+          <sbb-tab-group initial-selected-index="0">
+            <sbb-tab-label amount="16" icon-name="app-icon-small"> Tab title two </sbb-tab-label>
```
Comment (DavideMininni-Fincons) [inline src/elements/tabs/tab-group/tab-group.visual.spec.ts:74]:
```suggestion
            <sbb-tab-label amount="16" icon-name="app-icon-small"> Tab title one </sbb-tab-label>
```

File: src/elements/tabs/tab-group/tab-group.visual.spec.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+              <sbb-tab-label
+                amount=${numbersAndIcons ? 16 : nothing}
+                icon-name=${numbersAndIcons ? 'swisspass-small' : nothing}
+              >
+                Tab title two
+              </sbb-tab-label>
+
+              <sbb-tab-label
+                ?disabled=${true}
```
Comment (DavideMininni-Fincons) [inline src/elements/tabs/tab-group/tab-group.visual.spec.ts:51]:
```suggestion
                disabled
```

File: src/elements/tabs/tab-group/tab-group.visual.spec.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                Diam maecenas ultricies mi eget mauris pharetra et ultrices neque ornare aenean
+                euismod elementum nisi quis eleifend quam adipiscing vitae proin sagittis nisl
+                rhoncus mattis rhoncus urna neque viverra justo nec ultrices dui sapien eget mi
+                proin sed libero enim sed faucibus turpis in eu mi bibendum neque egestas congue.
+              </article>
+            </sbb-tab>
+
+            <sbb-tab-label amount="16" icon-name="swisspass-small"> Tab title two </sbb-tab-label>
+
+            <sbb-tab-label ?disabled=${true} amount="16" icon-name="train-small">
```
Comment (DavideMininni-Fincons) [inline src/elements/tabs/tab-group/tab-group.visual.spec.ts:86]:
```suggestion
            <sbb-tab-label disabled amount="16" icon-name="train-small">
```

File: src/elements/tabs/tab-group/tab-group.visual.spec.ts [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+      it(
+        visualDiffDefault.name,
+        visualDiffDefault.with(async (setup) => {
+          await setup.withFixture(html`
+            <sbb-tab-group size=${size} initial-selected-index="0">
+              <sbb-tab-label
+                amount=${numbersAndIcons ? 16 : nothing}
+                icon-name=${numbersAndIcons ? 'app-icon-small' : nothing}
+              >
+                Tab title two
```
Comment (DavideMininni-Fincons) [inline src/elements/tabs/tab-group/tab-group.visual.spec.ts:32]:
```suggestion
                Tab title one
```

File: src/elements/tabs/tab-group/tab-group.visual.spec.ts [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+              >
+                Tab title four
+              </sbb-tab-label>
+            </sbb-tab-group>
+          `);
+        }),
+      );
+    });
+
+    it(
```
Comment (DavideMininni-Fincons) [inline src/elements/tabs/tab-group/tab-group.visual.spec.ts:69]: What is this case covering exactly?

**Review by DavideMininni-Fincons — CHANGES_REQUESTED** [review]
Good work, i think a minimal test for the tab-component might be added too

File: src/elements/tabs/tab-group/tab-group.visual.spec.ts (hunk 6 above)
Comment (MarioCastigliano) [reply src/elements/tabs/tab-group/tab-group.visual.spec.ts:69]: if you mean the one starting at line 69 it's just the focused state

Comment (DavideMininni-Fincons) [reply src/elements/tabs/tab-group/tab-group.visual.spec.ts:69]: ah yes, i totally missed it

File: src/elements/tabs/tab-label/tab-label.visual.spec.ts [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+import '../tab.js';
+
+const cases = {
+  amount: ['', 16],
+  iconName: ['', 'app-icon-small'],
+};
+
+describe(`sbb-tab-label`, () => {
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    for (const state of visualDiffStandardStates) {
```
Comment (jeripeierSBB) [inline src/elements/tabs/tab-label/tab-label.visual.spec.ts:21]: As focus is part of the group, you could skip this state here

File: src/elements/tabs/tab-label/tab-label.visual.spec.ts [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+      it(
+        state.name,
```
Comment (jeripeierSBB) [inline src/elements/tabs/tab-label/tab-label.visual.spec.ts:23]: the active state in Firefox seems not to work, but I think we can accept this :)

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Review by DavideMininni-Fincons — APPROVED** [review]
LGTM

**Pull request merged**